pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/TopicStats.java

```diff
@@ -106,269 +112,270 @@ static void resetTypes() {
         metricWithTypeDefinition.clear();
     }
 
-    static void printTopicStats(SimpleTextOutputStream stream, String cluster, String namespace, String topic,
+    static void printTopicStats(Map<String, List<String>> metrics, String cluster, String namespace, String topic,
```

Comment:
it looks like this change is not only about the Function Worker.
Please update the title